### PR TITLE
fix(discover): Fix drilldowns when generating search conditions from array value fields

### PR DESCRIPTION
**Pull request by dashed**
Discover drilldowns need to support fields that can be array values.

### CODE REVIEW

**Review by wmak — APPROVED** [review]
For now I don't think we have a better way to deal with array fields. Can you add a comment explaining that where we add the values ORed together? Otherwise LGTM

File: src/sentry/static/sentry/app/views/eventsV2/utils.tsx [hunk 1]
```diff
@@ -353,6 +364,24 @@ function generateExpandedConditions(
...
+    if (Array.isArray(value)) {
+      parsedQuery.addOp('(');
+
+      const lastIndex = value.length - 1;
+
+      value.forEach((tagValue, index) => {
+        parsedQuery.addTagValues(key, [tagValue]);
+
+        if (lastIndex !== index) {
+          parsedQuery.addOp('OR');
```
Comment (Zylphrex) [inline src/sentry/static/sentry/app/views/eventsV2/utils.tsx:377]: I know we don't have array comparisons yet but let's use an `AND` here to be closer to array equality.

**Review by Zylphrex — APPROVED** [review]
LGTM for the interim until we have better support for arrays

File: src/sentry/static/sentry/app/views/eventsV2/utils.tsx [hunk 2]
```diff
@@ -353,6 +364,12 @@ function generateExpandedConditions(
   // Add additional conditions provided and generated.
   for (const key in conditions) {
     const value = conditions[key];
+
+    if (Array.isArray(value)) {
+      parsedQuery.addTagValues(key, value);
```
Comment (Zylphrex) [inline src/sentry/static/sentry/app/views/eventsV2/utils.tsx:369]: Why is this `addTagValues` vs `setTagValues`?

**Pull request merged**